Library/Styles/ButtonStyles.swift

```diff
@@ -15,10 +15,10 @@ public let applePayButtonStyle: ButtonStyle = { button in
 public let baseButtonStyle: ButtonStyle = { button in
   button
     |> roundedStyle(cornerRadius: Styles.grid(2))
-    |> UIButton.lens.titleLabel.font .~ UIFont.ksr_callout().bolded
-    |> (UIButton.lens.titleLabel .. UILabel.lens.textAlignment) .~ NSTextAlignment.center
-    |> UIButton.lens.contentEdgeInsets .~ .init(all: Styles.grid(2))
+    |> UIButton.lens.titleLabel .. UILabel.lens.font .~ UIFont.boldSystemFont(ofSize: 16)
```

Comment:
Lenses give us access to the `font` property of a UIButton. So this line could simply be `|> UIButton.lens.titleLabel.font .~ UIFont.boldSystemFont(ofSize: 16)` (this syntax is the most commonly used on our codebase)

Reply:
I thought we were moving away from lenses?

Reply:
Had to update `Prelude` so if you can also check it out and approve if everything looks fine